excercise-e-shop/assets/scss/page/home/_banner.scss

```diff
@@ -0,0 +1,87 @@
+.banner {
+  position: relative;
+  background: linear-gradient(180deg, rgba(0, 0, 0, 0.8) 0%, rgba(0, 0, 0, 0.5) 100%), url("../img/banner.png");
+  height: 800px;
+  width: 100%;
+  margin-bottom: 48px;
+  .container{
+    > ul {
+      position: absolute;
+      left: 0;
+      bottom: 0;
+      background-color: #fff;
+      width: 1100px;
+      padding: 38px 88px 24px 240px;
+      border-top-right-radius:70px;
+    }
+  }
```

Comment:
not clear, can replace ul by a class name.
we can remove `.container` which is not needed.